### PR TITLE
Remove approval step from sick leave

### PR DESCRIPTION
**Pull request by sponkrashin**
Issue #679

### CODE REVIEW

File: server/Arcadia.Assistant.Calendar.Notifications/SickLeaveAccountingEmailNotificationActor.cs [hunk 1]
```diff
@@ -17,55 +17,79 @@
...
+                        .PipeTo(this.Self);
+                    break;
 
-                    if (isProlonged && !dateChanged)
-                    {
-                        return;
-                    }
+                case CalendarEventChanged msg when
+                    msg.NewEvent.Type == CalendarEventTypes.Sickleave &&
+                    msg.NewEvent.Status == SickLeaveStatuses.Approved &&
```
Comment (santee) [inline server/Arcadia.Assistant.Calendar.Notifications/SickLeaveAccountingEmailNotificationActor.cs]: SickLeaveStatuses should not contain Approved anymore

Comment (sponkrashin) [reply server/Arcadia.Assistant.Calendar.Notifications/SickLeaveAccountingEmailNotificationActor.cs]: Ok then, what event statuses flow should be in this case?
It was Requested -> Approved (by manager) | Cancelled (by employee) -> Rejected (by manager) | Completed (by accounting).
After my changes it transformed to Requested -> Approved (automatically) | Cancelled (by employee) -> Cancelled (by employee) | Rejected (not used) | Completed (by accounting).

Comment (santee) [reply server/Arcadia.Assistant.Calendar.Notifications/SickLeaveAccountingEmailNotificationActor.cs]: Requested (can be prolonged) -> Cancelled (by employee) / Completed (by accounting). You may rename Requested to 'Not Completed' if requested/approved confuse you.

Comment (santee) [reply server/Arcadia.Assistant.Calendar.Notifications/SickLeaveAccountingEmailNotificationActor.cs]: Automatic approval is weird.

Comment (sponkrashin) [reply server/Arcadia.Assistant.Calendar.Notifications/SickLeaveAccountingEmailNotificationActor.cs]: Done.

File: server/Arcadia.Assistant.CSP/SickLeaves/CspSickLeavesRegistry.cs [hunk 2]
```diff
@@ -99,50 +94,6 @@ protected override void OnReceive(object message)
                     this.ScheduleNextDatabaseRefresh();
                     break;
 
-                case RefreshDatabaseCreatedSuccess msg:
-                    this.logger.Debug($"Sick leave with id {msg.Event.EventId} for employee {msg.Event.EmployeeId} " +
-                        "was added to CSP database manually");
-
-                    Context.System.EventStream.Publish(new CalendarEventCreated(msg.Event, msg.CreatedBy, msg.Timestamp));
```
Comment (santee) [inline server/Arcadia.Assistant.CSP/SickLeaves/CspSickLeavesRegistry.cs]: we don't need to publish Created event anymore?

Comment (sponkrashin) [reply server/Arcadia.Assistant.CSP/SickLeaves/CspSickLeavesRegistry.cs]: It is published in the line 187.

**Pull request merged**